template/src/store/createStore.js

```diff
@@ -0,0 +1,17 @@
+import { createStore, compose } from 'redux'
```

Comment:
It doesn't seem like we are using compose right? We use the one from DEVTOOLS instead?